.github/workflows/.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+    "extends": ["stylelint-config-standard"],
+    "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+    "rules": {
+      "at-rule-no-unknown": null,
+      "scss/at-rule-no-unknown": [
+        true,
+        {
+          "ignoreAtRules": [
+            "tailwind",
+            "apply",
+            "variants",
+            "responsive",
+            "screen"
+          ]
+        }
+      ]
+    },
+    "csstree/validator": true,
+    "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css"]
+  }
+  
```

Comment:
- [ ] As specified by the previous reviewer to resolve all linters errors, as I can see you did your best but the problem is actually this `stylelintrc.json` file inside the `workflows`. Kindly put it at the `root` path outside the `.github/workflows`. Because it is inside the `github/workflows`. The GitHub actions consider it as part of linter checking and that's the reason your `linters` are not working properly.